service/telemetry/tracer.go

```diff
@@ -48,9 +49,10 @@ func attributes(set Settings, cfg Config) map[string]interface{} {
 
 // New creates a new Telemetry from Config.
 func newTracerProvider(ctx context.Context, set Settings, cfg Config) (trace.TracerProvider, error) {
-	if globalgates.NoopTracerProvider.IsEnabled() {
+	if globalgates.NoopTracerProvider.IsEnabled() || cfg.Traces.Level == configtelemetry.LevelNone {
```

Comment:
Should it be documented somewhere that level:none breaks zpage?

Reply:
I can add a note in the zpages documentation